docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -0,0 +1,135 @@
+# Creating a Simple Cross-Platform Binary Module with the Dotnet CLI
+
+This example shows how you can create a simple, binary module using the dotnet
+CLI (command line interface).  For this example, you can use any OS that the
+.NET Core 2.0 SDK installs on.  The Windows for Subsystem Linux console works
+well for this.
+
+Because the module's assembly will be created as a .NET Standard 2.0 class library,
+the same assembly can be loaded in Windows PowerShell as well as PowerShell Core.
+This means you don't have to build separate assemblies to target these two different
+implementations of PowerShell.
+
+## Prerequisites
+* Download and install the [.NET Core 2.0 SDK][net-core-sdk] for your operating system.
+  On Linux, I recommend using a package manager to install the SDK.
```

Comment:
Probably change to `It is recommended to use a package manager to install the SDK on Linux` or something similar. Documentation should not use first person.

Reply:
fixed